examples/with-tailwindcss-emotion/pages/_app.js

```diff
@@ -1,13 +1,20 @@
 import Head from 'next/head'
-import '../styles/base.css'
+import { Global } from '@emotion/react'
+import xw from 'xwind'
```

Comment:
This change is not required. We recommend having the global CSS with a import in `_app.js` when possible.

Reply:
Alright, I've reverted the removal of the `base.css` file. (This file can be generated with the "build:base-css" script)